sale_order_line_sequence/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_sale_order_line_sequence
```

Comment:
**[change]** Missing license header

`# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).`

Reply:
@BT-crodriguez License headers are not needed in `__init__.py` files.